src/components/IOBanner.js

```diff
@@ -70,7 +70,7 @@ const IOBanner = ({ isMobile }) => {
               font-size: 0.65rem;
             }
           `}
-          to="https://www.youtube.com/watch?v=jN3Bk473R4w"
+          to="https://docs.newrelic.com/docs/full-stack-observability/observe-everything/get-started/new-relic-quickstarts-overview/"
```

Comment:
This page is throwing a `404`, is it being launched tomorrow along with everything else?

Reply:
Yes, @bradleycamacho is responsible to make that page live on 9/29, good question

Reply:
Yup! I'm merging it in a few hours before the I/O launch, so it should stop 404ing a bit before launch.